tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -314,50 +314,15 @@
               "max_counted": null,
               "min_needed": null,
               "explanation": null,
-              "dist_req": "EC"
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
-              "dist_req": "EM"
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
+              "dist_req": ["EC", "EM", "HA", "LA", "SA"],
```

Comment:
Could you format this with one item per line (see the other lists in this file for reference)?